### PR TITLE
Support for Idealmedia

### PR DESCRIPTION
**Pull request by DenyHF**

### CODE REVIEW

**Comment by torch2424** [comment]
@DenyHF Thank you very much for making the changes!

So everything looks good in the code, as well as the example:

Screen Shot 2019-05-06 at 1 09 35 PM

However, when I added the following log statement:

Screen Shot 2019-05-06 at 1 04 54 PM

Seems like the `renderStart_` in `amp-ad-xorigin-iframe-handler.js` never gets called even though you have `renderStartImplemented: true`. Is this done on the server side? Or is it something on my end?

Thank you!

**Comment by DenyHF** [comment]
@torch2424 
Hello, this problem is on our side.
I deleted this parameter from the config

File: ads/_config.js [hunk 1]
```diff
@@ -498,6 +498,8 @@ export const adConfig = {
 
   'ibillboard': {},
 
+  'idealmedia': {},
```
Comment (torch2424) [inline ads/_config.js]: So I noticed here you removed implementing `renderStartImplemented`. If possible, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

If it's something you plan on supporting, and it's a server-side thing. I'm okay with merging this, and you can fix on your server 😄 

Let me know what you think! 👍

Comment (DenyHF) [reply ads/_config.js]: Hello, at the moment we do not support this method.
We will implement it soon.
revert back to `renderStartImplemented=true` ?

Comment (torch2424) [reply ads/_config.js]: By @lannka suggestion, let's set this to `renderStartImplemented: false`, and then we can merge this in. Once you implement on the server, we can then open a new PR for `renderStartImplemented: true` 😄 👍

**Comment by torch2424** [comment]
@DenyHF Thank you very much for making these changes!

Merging! 👍

**Comment by torch2424** [comment]
@DenyHF Fixed a super small merge conflict as well with your other PR. Thanks again! Will merge once the tests pass (I got it from here) 😄

**Pull request merged**